src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -11,67 +11,100 @@ import askgrok from 'images/grok_screenshot-crop_ask-grok.webp'
 
 <img
         title="Screenshot showing the Ask Grok button in the New Relic platform"
         alt="ask grok button"
         src={askgrok}
     />
 
-[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) is a powerful tool you can use to gain insights into systems, but when you're in a hurry you might not have time to create  the right query to get the data you need. If you're struggling to get a query to work, ask our AI to help you. You can ask questions like, "How is this service performing?" or "How has this service been performing over the past 24 hours?"
 
 To enable NRAI in the NRQL console: 
 
 1. Go to the user menu at the bottom left of the New Relic UI.
 2.  Click the **Features** tab.
 3. Scroll down to the **New Relic AI** section and click **Enable**.
 
-## Sample questions [#sample-questions] 
+### Get help analyzing error logs [#errors]
 
-You can chat with our AI to better understand every aspect of your data but if you're struggling with where to begin, here are some prompts. 
+Error log lines can be difficult to parse. New Relic AI helps you learn more about errors, query for related information, and ask about next steps. 
 
-### Get started with New Relic 
-Our AI chat experience can also help get you started using New Relic. If you need help organizing your data and setting up alerts, NRAI can act as a guide throughout your onboarding process. Ask our AI questions like:
+<img
+    title="explain this error"
+    alt="A screenshot depicting the explain this error feature in New Relic Logs"
+    src={aiNewRelicAiExplainThisError}
+/>
 
-* How can I use Terraform to create dashboards? 
-* How can I improve my alerts setup?
-* How can I instrument AWS?
+<figcaption>
+    To ask our AI about an error in your logs Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities)** > **Logs** > **All logs** >  and search for `errors`.
+</figcaption>
 
-### Improve response time
-If you want to get to the root cause of latency issues for your app, here are some good questions to ask:
+To chat with New Relic AI about log errors:
 
-* What are some recent incidents in my system?
-* Which infrastructure hosts are showing signs of degradation over the past hour?
+1. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities)** > **Logs**.
+2. Click **All logs**.
+3. Search for `errors`.
+2. Click on any resulting log lines and look for the **Explain this error** button at the top of the log details panel. Our AI will provide a summary of the error. 
 
-### Take observability to the next level 
-Our AI can also be your partner in observability maturity. If you're responsible for ensuring that your app can handle spikes for a big e-commerce event like Black Friday, it's important to understand the state of your system so you can improve it _ahead of time_. Ask our AI:
```

Comment:
@homelessbirds since "Black Friday" is only in the United States, maybe you should mention that to help set the context for non-US readers.